### PR TITLE
Rearranging BBE and wordings

### PR DESCRIPTION
**Pull request by wggihan**
Fixes #<Issue Number>

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Required Balo version update
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

**Comment by wggihan** [comment]
> Regarding changing basic to simple Should we change the file names as well?

IMO, that's not mandatory. Here the main purpose is to provide unified naming convention in the example menu in the Bio.

**Comment by riyafa** [comment]
> > Regarding changing basic to simple Should we change the file names as well?
> 
> IMO, that's not mandatory. Here the main purpose is to provide unified naming convention in the example menu in the Bio.

I mean when they open the example there would be an inconsistency in the file name used. May be we can fix that later.

File: examples/index.json [hunk 1]
```diff
@@ -992,31 +972,31 @@
         "category": "Working over the network",
         "samples": [
             {
-                "name": "Artemis ANYCAST consumer",
+                "name": "ANYCAST consumer",
```
Comment (praneesha) [inline examples/index.json]:
```suggestion
                "name": "ANYCAST Consumer",
```
Shouldn't it be in uppercase to be consistent with the rest (e.g., "Basic Consumer" above).

File: examples/index.json [hunk 2]
```diff
@@ -992,31 +972,31 @@
...
                 "url": "artemis-anycast-session-consumer"
             },
             {
-                "name": "Artemis ANYCAST producer",
+                "name": "ANYCAST producer",
```
Comment (praneesha) [inline examples/index.json]:
```suggestion
                "name": "ANYCAST Producer",
```
Shouldn't it be in uppercase to be consistent with the rest (e.g., "Basic Consumer" above).

File: examples/index.json [hunk 3]
```diff
@@ -1035,6 +1015,32 @@
             }
         ]
     },
+    {
+        "title": "Access Control",
+        "column": 3,
+        "category": "Working over the network",
```
Comment (praneesha) [inline examples/index.json]:
```suggestion
        "category": "Working via the network",
```

File: examples/index.json [hunk 4]
```diff
@@ -1035,6 +1015,32 @@
...
+        "samples": [{
+                "name": "Secured Service With JWT",
```
Comment (praneesha) [inline examples/index.json]:
```suggestion
                "name": "Secured Service with JWT",
```

Normally, we don't capitalize conjunctions.

File: examples/index.json [hunk 5]
```diff
@@ -1035,6 +1015,32 @@
...
+                "url": "secured-service-with-jwt"
+            },
+            {
+                "name": "Secured Service With Basic Auth",
```
Comment (praneesha) [inline examples/index.json]:
```suggestion
                "name": "Secured Service with Basic Auth",
```

Normally, we don't capitalize conjunctions.

File: examples/index.json [hunk 6]
```diff
@@ -1035,6 +1015,32 @@
...
+                "url": "secured-service-with-basic-auth"
+            },
+            {
+                "name": "Secured Client With Basic Auth",
```
Comment (praneesha) [inline examples/index.json]:
```suggestion
                "name": "Secured Client with Basic Auth",
```

Normally, we don't capitalize conjunctions.

File: examples/index.json [hunk 7]
```diff
@@ -1035,6 +1015,32 @@
...
+                "url": "secured-client-with-basic-auth"
+            },
+            {
+                "name": "Secured Client With JWT Auth",
```
Comment (praneesha) [inline examples/index.json]:
```suggestion
                "name": "Secured Client with JWT Auth",
```

Normally, we don't capitalize conjunctions.

File: examples/index.json [hunk 8]
```diff
@@ -1035,6 +1015,32 @@
...
+                "url": "secured-client-with-jwt-auth"
+            },
+            {
+                "name": "Secured Client With OAuth2",
```
Comment (praneesha) [inline examples/index.json]:
```suggestion
                "name": "Secured Client with OAuth2",
```

Normally, we don't capitalize conjunctions.

File: examples/index.json (hunk 3 above)
Comment (wggihan) [reply examples/index.json]: That is the main category name. So won't apply this change.

File: examples/index.json [hunk 9]
```diff
@@ -969,7 +949,7 @@
         "column": 3,
         "category": "Working over the network",
         "samples": [{
-            "name": "NATS Basic Example",
+            "name": "Basic Publisher Subscriber",
```
Comment (MaryamZi) [inline examples/index.json]: Should this be "Basic Publisher and Subscriber" or "Basic Publish-Subscribe"?

Comment (wggihan) [reply examples/index.json]: "Basic Publisher and Subscriber" more suitable since we have two different files IMO.

**Pull request merged**